app/models/patient.rb

```diff
@@ -147,6 +147,8 @@ def self.pledged_status_summary(line)
     patients = Patient.in(line: line)
                       .where(:fund_pledge.nin => [0, nil, ''])
                       .where(:fund_pledged_at.gte => start_of_week)
+                      .where(:resolved_without_fund.nin => [true])
```

Comment:
Any reason we're not just saying `in`? The negation term combined with `without` is a bit hard to parse, and I can see reading this line a bit too fast as the opposite with how close `nin` and `in` are. I might be being a bit nitpicky though. Thoughts?